Makefile

```diff
@@ -0,0 +1,44 @@
+XCARGS := -workspace VelocidiSDK.xcworkspace \
```

Comment:
Please add this file to the xcode group, so that it is visible inside the project.

Reply:
Done in 323259f